Update Ambari docs for new features

wolf31o2 opened the pull request:
This removes documentation about features which were implemented during the release 3.5 timeline.

wolf31o2 committed:
Update Ambari docs for new features

wolf31o2 committed:
Add Auth Server to list of master services

wolf31o2 committed:
Remove section about not supported

Commit:
Update ambari.rst

Minor edits.

Commit:
Add Ambari HA Installation instructions

awholegunch commented:
Running as [Quick Build 1](http://builds.cask.co/browse/CDAP-DQB104-1). Fails, but unrelated to this PR.

Page of interest: 
- [Installation using Apache Ambari](http://builds.cask.co/artifact/CDAP-DQB104/shared/build-1/Docs-HTML/3.5.0/en/admin-manual/installation/ambari.html)
- [Security](http://builds.cask.co/artifact/CDAP-DQB104/shared/build-1/Docs-HTML/3.5.0/en/admin-manual/installation/ambari.html#enabling-security)
- [HA Installation](http://builds.cask.co/artifact/CDAP-DQB104/shared/build-1/Docs-HTML/3.5.0/en/admin-manual/installation/ambari.html#enabling-cdap-ha)

wolf31o2 commented on cdap-docs/admin-manual/source/installation/ambari.rst:
```diff
@@ -355,7 +349,70 @@ Kerberos support in CDAP is automatically enabled when enabling Kerberos securit
+
+- Using the Ambari UI, add additional hosts for the ``CDAP Gateway/Router
+  Service`` to additional machines.
+- Start each ``CDAP Gateway/Router Service`` role.
```
It's just `CDAP Router` in Ambari. I don't mention Gateway.

awholegunch replied:
Fixed.

wolf31o2 commented:
We can also remove the section about SPARK_HOME (Customize CDAP 6) and Explore (Customize CDAP 8) since we always require Hive (no optional dependencies in Ambari)...

Commit:
Update ambari.rst

Remove "Gateway"

Commit:
Update ambari.rst

Remove the section about SPARK_HOME (Customize CDAP 6) and Explore (Customize CDAP 8) since we always require Hive (no optional dependencies in Ambari)...

awholegunch commented:
Removed sections in CDAP no. 6 and CDAP no. 8. Please check my last two commits. LGTM.

wolf31o2 commented:
It looks like there may still be screenshots for these. Are you planning on just updating those with another PR, since you are going to be doing screenshots, anyway? If so, this 👍 LGTM

awholegunch commented:
Yes, the screenshots are coming along elsewhere.
